src/index.js

```diff
@@ -372,4 +384,31 @@ Dropzone.defaultProps = {
   minSize: 0
 };
 
+Dropzone.propTypes = {
```

Comment:
Hmm, the whole point of having Flow types is to remove PropTypes. I'm wondering why did you add them back?